request/graphql/schema/types/commits.go

```diff
@@ -47,47 +47,47 @@ var (
 		Name:        request.CommitTypeName,
 		Description: commitDescription,
 		Fields: gql.Fields{
-			"height": &gql.Field{
+			request.HeightFieldName: &gql.Field{
```

Comment:
praise: this is much cleaner

Reply:
It looks nicer, but now we have lost protection against these changing.  I have mixed feelings about this change.